tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/matters.controller.js

```diff
@@ -836,7 +836,32 @@
             }
             //#endregion
 
-
+            vm.showMatterAsPinOrUnpin = function (response, searchRequest) {

```

Comment:
Added one reusable method which will be called at several places for pin and unpin functionality